Python/Product/VSCode/AnalysisVsc/Commands/Controller.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.DsTools.Core.Services;
```

Comment:
Can we rename this namespace (wherever it came from)?

Reply:
It used to be Microsoft.R.Common.Core. Basically various useful extensions etc. How do we want it named?